app/controllers/questions_controller.rb

```diff
@@ -0,0 +1,40 @@
+class QuestionsController < ApplicationController
+  before_action :find_test, :find_question
+
+  rescue_from ActiveRecord::RecordNotFound, with: :rescue_with_question_not_found
+
+  def index; end
+
+  def show; end
+
+  def new; end
+
+  def create
+    @test.questions.create!(question_params)
+    redirect_to test_questions_path(@test)
+  end
+
+  def destroy
+    test_id = @question.test_id
+    @question.destroy
+    redirect_to test_questions_path(test_id)
```

Comment:
```suggestion
    redirect_to test_questions_path(@question.test)
```
объект `@question` будет доступен в памяти после удаления его из базы, первую строку метода можно убрать (присвоение "тест-айди"